test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -420,22 +420,32 @@ class SystemInfo:
     """
     logger = logging.getLogger(__name__)
 
-    def __init__(self, hostname=None):
+    def get_system_info(self, hostname=None):
+        """
+        used to get system's ram size and disk size information.
+
+        :system_ram: Available ram(in GB) of the test running machine
```

Comment:
system_ram and system_disk are not the arguments to this function, document 'hostname' instead.
Also, the docstring for params is written as:
```
:param <param name>: <description>
:type <param name>: <type>
```